client/src/actions/authActions.js

```diff
@@ -7,8 +7,18 @@ import {
   SET_USER,
 } from "constants/action-types";
 
+import TagManager from "react-gtm-module";
+
 const GET_CURRENT_USER_ENDPOINT = "/api/users/current";
 
+const getTagManagerArgs = (userId) => {
+  return {
+    dataLayer: {
+      userId: userId,
+    },
+  };
+};
+
```

Comment:
Can just use implicit return here and get rid of one layer of brackets. 
```
const getTagManagerArgs = (userId) => {
    dataLayer: {
      userId: userId,
    },
};
```

Are you still planning to change the name as we discussed?

Reply:
No naming, just implicit sending the userId :)